apps/meteor/tests/unit/server/federation/infrastructure/rocket-chat/hooks/hooks.spec.ts

```diff
@@ -24,117 +26,155 @@ const { FederationHooks } = proxyquire
 				priority: { HIGH: 'high' },
 				remove,
 				add: (_name: string, callback: (...args: any[]) => void, _priority: string, _id: string) => {
+					console.log('ADD fuck');
```

Comment:
Do we need this console.log?

Reply:
nop, my bad I was forced to debug because this injection creates two instances, I though my rebase had removed